src/elements/content-preview/ContentPreview.js

```diff
@@ -741,6 +753,11 @@ class ContentPreview extends React.PureComponent<Props, State> {
         this.preview.addListener('preview_metric', this.onPreviewMetric);
         this.preview.addListener('thumbnailsOpen', () => this.setState({ isThumbnailSidebarOpen: true }));
         this.preview.addListener('thumbnailsClose', () => this.setState({ isThumbnailSidebarOpen: false }));
+
+        if (previewOptions.showAnnotationsControls) {
+            this.preview.addListener('annotationCreate', onAnnotationCreate);
```

Comment:
Do we have to do handle removing this listener?

Reply:
I don't think so because when preview is destroyed, we do `preview.removeAllListeners()` which should take care of it. But we may need to add some handling when we care about the annotator instance in the future. Thanks for bringing this up